apollo-router/src/axum_http_server_factory.rs

```diff
@@ -3221,12 +3221,11 @@ Content-Type: application/json\r
                     },
                     "errors": [
                         {
-                            "message": "invalid content: Missing key `_entities`!",
-                            "path": ["topProducts", "@"],
-                            "extensions": {
-                                "type": "ExecutionInvalidContent",
-                                "reason": "Missing key `_entities`!"
-                            }
+                            "message": "couldn't find mock for query {\"query\":\"query TopProducts__reviews__1($representations:[_Any!]!){_entities(representations:$representations){...on Product{reviews{__typename id product{__typename upc}}}}}\",\"operationName\":\"TopProducts__reviews__1\",\"variables\":{\"representations\":[{\"__typename\":\"Product\",\"upc\":\"1\"},{\"__typename\":\"Product\",\"upc\":\"2\"}]}}"
+                        },
+                        {
+                            "message": "Missing key `_entities`!",
```

Comment:
Is it possible for us to have something that the user can act on?

Subgraph response from `<subgraph>` was missing key `_entities`!

Reply:
sure! [c380f92](https://github.com/apollographql/router/pull/1870/commits/c380f9289c798938227e0dba2290a3e2286c5634)